runtime/runtime-core/common/src/aws/smithy/kotlin/runtime/businessmetrics/BusinessMetricsUtils.kt

```diff
@@ -74,7 +74,7 @@ public enum class SmithyBusinessMetric(public override val identifier: String) :
     RETRY_MODE_ADAPTIVE("F"),
     GZIP_REQUEST_COMPRESSION("L"),
     PROTOCOL_RPC_V2_CBOR("M"),
-    SERVICE_ENDPOINT_OVERRIDE("N"),
-    ACCOUNT_ID_BASED_ENDPOINT("O"),
-    SIGV4A_SIGNING("P"),
+    ENDPOINT_OVERRIDE("N"),
+    ACCOUNT_ID_ENDPOINT("O"),
```

Comment:
Can we also emit the new `ACCOUNT_ID_MODE_PREFERRED`, `ACCOUNT_ID_MODE_DISABLED`, `ACCOUNT_ID_MODE_REQUIRED` metrics?

Reply:
Yeah, we can add that